backend/api/src/main/java/com/virtuslab/gitmachete/backend/api/IRootManagedBranchSnapshot.java

```diff
@@ -17,4 +17,9 @@ default IRootManagedBranchSnapshot asRoot() {
   default INonRootManagedBranchSnapshot asNonRoot() {
     throw new NotImplementedError();
   }
+
+  @Override
+  default boolean isTracked() {
+    throw new NotImplementedError();
```

Comment:
🤔

Reply:
Shouldn't rebase onto remote tracking branch be allowed for a root branch as well?